### PR TITLE
chore(merge): No PDF - Merge `main` into `main-application-no-pdf`

### PR DESCRIPTION
**Pull request by ttbarnes**
## Introduction :pencil2:
This PR merges the `main` branch into `main-application-no-pdf`.

This prepares us for merging `main-application-no-pdf` into `main`.

## Resolution :heavy_check_mark:
- Merge `main` into `main-application-no-pdf`.

### CODE REVIEW

File: .github/workflows/infrastructure.yml [hunk 1]
```diff
@@ -1,23 +1,32 @@
...
+# The workflow is triggered on pushes to the 'infrastructure' and 'EMS-2898-database-tier' branches. It uses environment variables to define the product, environment,
+# timezone, and target. The jobs are organized into sections for setup, base infrastructure creation, security configuration, web app configuration, logging and diagnostic settings, and health check alerts.
+# Each step in the workflow performs a specific task using the Azure CLI, such as creating resource groups, app service plans, log analytics workspaces, container registries,
+# virtual networks, subnets, VNET peering, databases, web apps, front doors, WAF policies, private endpoints, DNS configurations, and more.
+# The workflow also includes conditional steps based on the target environment, such as different configurations for pre-production and production environments.
+# Overall, this code snippet provides a comprehensive configuration for setting up infrastructure and configuring various resources in an Azure environment using GitHub Actions.
 #
-# Naming conventions
```
Comment (Zainzzkk) [inline .github/workflows/infrastructure.yml:7]: should this be removed?

File: src/ui/public/js/mojFrontend.js
Comment (Zainzzkk) [inline src/ui/public/js/mojFrontend.js:1]: hasn't this been removed?

Comment (ttbarnes) [reply src/ui/public/js/mojFrontend.js:1]: Good spot, removed

**Review by Zainzzkk — APPROVED** [review]

File: .github/workflows/infrastructure.yml [hunk 2]
```diff
@@ -1,23 +1,32 @@
-# Introduction
```
[the remark this answers is not in the document]
Comment (ttbarnes) [reply .github/workflows/infrastructure.yml:1]: Good spot - discussed, we should go with the latest updated documentation which is in the `main` branch; This is the most recent version: https://github.com/UK-Export-Finance/exip/commit/e2eb580fe610938da969e335185174493c56b737

File: .github/workflows/infrastructure.yml (hunk 1 above)
Comment (ttbarnes) [reply .github/workflows/infrastructure.yml:7]: Good spot - discussed, we should go with this version, which is the latest updated documentation in the `main` and `no-pdf` branches. This is the most recent version: https://github.com/UK-Export-Finance/exip/commit/e2eb580fe610938da969e335185174493c56b737

**Pull request merged**